hyperdb/hyperdb.py

```diff
@@ -1,7 +1,7 @@
 import json
 import numpy as np
 import openai
-from hyperdb.galaxy_brain_math_shit import hyper_SVM_ranking_algorithm_sort
+from hyperdb.galaxy_brain_math_shit import *
```

Comment:
Dear [insert name of offender],

I would like to take a moment to talk to you about your reckless use of import * in Python. Your lackadaisical approach to importing modules is not only harmful to your code, but also to the delicate sensibilities of those of us who care deeply about the craft of software development.

Let me begin by saying that import * is the equivalent of bringing a chainsaw to a delicate surgery. Sure, it may seem like a good idea at first, but when things start going wrong, you're going to wish you had taken a more cautious approach. Think of import * as a gateway drug to sloppy, unmaintainable code that will haunt you for the rest of your life.

The thing is, importing everything from a module is like opening up a Pandora's Box of chaos. You're bringing in all sorts of variables, functions, and classes without really understanding how they all work together. It's like inviting a bunch of strangers into your code and hoping they'll all get along. Spoiler alert: they won't.

And let's not forget about the cognitive overhead of import *. When you're working on a large codebase, it's important to know exactly what you're working with. When you import everything, it becomes much harder to keep track of what's going on. It's like trying to find a needle in a haystack, except the haystack is on fire and the needle is also on fire.

Now, I know what you're thinking. "But [insert name], importing everything is so much easier!" Yes, it may seem easier in the short term, but trust me, it's not worth it in the long run. You're setting yourself up for a world of hurt down the line.

So, please, for the love of Guido van Rossum, stop using import *. Take a little time to figure out exactly what you need, and only import those things. Your future self (and your coworkers) will thank you.

Sincerely,
[Insert your name here]

Reply:
Nice use of ChatGPT to write ridiculously exaggerated comments.

Try to understand the structure of the repo before you start commenting on PRs.

I'm aware that import * is not a good practice in general, but in this case I would argue that it makes sense because galaxy_brain_math_shit only contains helper math functions which needs to be in the hyperDB namespace. 

smh